kolena/dataset/asset.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from kolena.workflow.asset import BaseVideoAsset
+from kolena.workflow.asset import BinaryAsset
+from kolena.workflow.asset import ImageAsset
+from kolena.workflow.asset import PlainTextAsset
+from kolena.workflow.asset import PointCloudAsset
+from kolena.workflow.asset import VideoAsset
+
+__all__ = ["ImageAsset", "PlainTextAsset", "BinaryAsset", "PointCloudAsset", "BaseVideoAsset", "VideoAsset"]
```

Comment:
Are we intentionally skipping `Annotation` and `Asset`?

Reply:
i just use `_ASSET_TYPES` and `_ANNOTATION_TYPES`, i don't think we expect users will need to inherit the base class